ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -2311,6 +2311,7 @@ def flatten(*, backend: str, ret):
                     x, dtype=ivy_backend.Dtype(str(np.asarray(x).dtype))
                 )
                 for x in ret_flat
+                if ivy.is_ivy_array(x)
```

Comment:
Hi @rohitkg83, these changes you made in `function_testing.py` result in functions not being tested properly. Please remove them.
Because `einsum_path`'s return is unusual, we will have to manually check it's correct.

Reply:
Hey @AnnaTz  - I have removed these changes but the tests were still failing as these were trying to flatten the return with one return being string. I have made some changes to function testing file wherein it will only flatten the return if test values is true. The reason being when it is false we are returning non-flattened ret and front-end ret. Hope it makes sense!